tests/ttnn/unit_tests/operations/test_new_conv2d.py

```diff
@@ -370,12 +372,15 @@ def test_conv_ws(
     stride_h = stride
     stride_w = stride
     batch_size = 2
-    fp32_accum = False
-    packer_l1_acc = False
+    fp32_accum = True
```

Comment:
This is not OK. 
This way again you are leaving big code paths untested, you need to test both fp32 on/off and both l1 acc on/off

Reply:
Adding this functionality would make this test suite take a very long time to run. Additionally, testing all these different options against every input combination may not add much value.

Reply:
This what we already do for HS test cases (test all params variants) and it seems fine?

Reply:
please test for both cases, otherwise as Pavle pointed out, we will have untested paths in the width sharding case.

Reply:
I've added a new[ test case ](https://github.com/tenstorrent/tt-metal/blob/339d2c80116b610696aa94e961fb31af0574f44d/tests/ttnn/unit_tests/operations/test_new_conv2d.py#L360)to test all possible code paths.